src/uct/ib/dc/dc_mlx5.h

```diff
@@ -160,12 +160,28 @@ typedef enum {
 } uct_dc_tx_policy_t;
 
 
+/**
+ * lag_tx_port_affinity config for DCIs
+ * - auto: auto detect lag mode to set lag_tx_port_affinity or not
+ * - affinity:
+ *   set lag_tx_port_affinity in lag mode
+ * - hash:
+ *   do not set lag_tx_port_affinity in lag mode
```

Comment:
pls move description of values directly to enum and use doxy `@ref` if you want to reference to exact field:
```
/**
 * automalically detect lag mode
 */
UCT_DC_MLX5_LAG_PORT_AUTO,

/**
 * set @ref <reference to lag_tx_port_affinity field> in lag mode
 */
UCT_DC_MLX5_LAG_PORT_AFFINITY,
...
```

Reply:
Done